README.md

```diff
@@ -135,3 +135,20 @@ await user.delete();
 // Using an object's id
 await xata.db.users.delete('rec_1234abcdef');
 ```
+
+## Deno support
```

Comment:
```suggestion
## Deno support
```

There's a weird whitespace that prevents this line to be rendered as a heading. See https://github.com/xataio/client-ts/blob/f1e660da2d79ab2314266f0fb2e18055e944bbe2/README.md